api/envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -162,6 +162,6 @@ message ResponseFlagFilter {
   // This field is optional. If it is not specified, then any response flag will pass
   // the filter check.
   repeated string flags = 1 [(validate.rules).repeated .items.string = {
-    in: ["LH", "UH", "UT", "LR", "UR", "UF", "UC", "UO", "NR", "DI", "FI", "RL", "UAEX", "RLSE"]
+    in: ["LH", "UH", "UT", "LR", "UR", "UF", "UC", "UO", "NR", "DI", "FI", "RL", "UAEX"]
```

Comment:
Can we just avoid reverting anything in api/?

Reply:
@htuch sure. I removed this because it would not create a proto ordering problem. But I see the benefit of preserving anything in API.

Reply:
nevermind, I forgot that I made these tests pretty robust a couple months ago, so adding the flag here makes the test code detect that we have added a flag to the proto that we haven't supported in the code. So, in the end, I need to leave this out in the revert.